C/CUTEst/build_tarballs.jl

```diff
@@ -0,0 +1,128 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CUTEst"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "https://github.com/ralna/ARCHDefs/archive/generic-system.zip" =>
```

Comment:
Why these aren't separate packages?

Reply:
They're never used separately. They're mostly separate for ease of maintenance.

Reply:
Anyway this needs to be a permanent link, otherwise a new commit to the `generic-system` branch will break this source, as the hash would be different:
```suggestion
    "https://github.com/ralna/ARCHDefs/archive/646f7c7b3848a547a70672243c1d75421268a8a8.zip" =>
```